src/components/post-include-controls.js

```diff
@@ -40,9 +45,31 @@ export const PostIncludeControls = ( { attributes, setAttributes } ) => {
 				[]
 			);
 		},
-		[ postType, multiplePosts, excludeCurrent ]
+		[ postType, multiplePosts, excludeCurrent, searchArg ]
 	);
 
+	console.log( posts );
```

Comment:
Not sure if you're done with the changes here but let's be sure to remove these console.logs before we merge :)